docs/backup_restore_guide.md

```diff
@@ -0,0 +1,69 @@
+# Harbor Backup and Restore Guide
+
+When the container registry storage is local file system, the straightforward way to backup a standalone Harbor data is to backup the /data directory and recover it to /data directory.
+For some special cases, if you want more efficient way to backup the Harbor data, you can refer the guide following. 
+
+## Backup Harbor data
+
+1. Login to the Harbor host, find a disk which has enough space, create a backup directory in it, for exmaple, /backup
+2. Download script from https://github.com/goharbor/harbor/blob/backup_restore/tools/harbor-backup.sh and copy it to /backup
+3. Shutdown the Harbor instance  
+```
+docker-compose down -v
+```
+4. Backup data
+```
+cd /backup
+ls
+harbor-backup.sh
+```
+
+Stop all running containers. Backup all data
```

Comment:
the command harbor-backup.sh is listed twice. before this step and here. do they need to run it twice?

is it possible to take a good backup without downtime of the Harbor services?